backend/src/main/java/ca/mcgill/purposeful/model/CollaborationConfirmation.java

```diff
@@ -0,0 +1,64 @@
+package ca.mcgill.purposeful.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.Id;
+import org.hibernate.annotations.GenericGenerator;
+
+/**
+ * The CollaborationConfirmation class, the model for regular user's collaboration confirmation to a
+ * request in the database
+ */
+@Entity
+public class CollaborationConfirmation {
+
+  // ------------------------
+  // CollaborationConfirmation Attributes
+  // ------------------------
+
+  @Id
+  @GeneratedValue(generator = "uuid")
+  @GenericGenerator(name = "uuid", strategy = "uuid2")
+  private String id;
+
+  @Column(nullable = true)
```

Comment:
if you make changes in the other PR, hopefully these ones won't override them? I try make sure they don't

Reply:
They shouldn't override, but some things will stop working, so I'll rework it a bit after the other pr gets merged

Reply:
Other PR merged if you want to do that :)